hack/update-api-reference-docs.sh

```diff
@@ -37,12 +47,40 @@ echo "Reading swagger spec from: ${SWAGGER_PATH}"
 mkdir -p $V1_PATH
 mkdir -p $V1BETA1_PATH
 
-docker run --rm -v $V1_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v3 \
+docker run -u $(id -u) --rm -v $V1_TEMP_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v4 \
```

Comment:
Ah, well there's your problem. The path that you pass to `docker run -v PATH:/output` will get resolved on the host, not the container. So it's actually writing to `${KUBE_TEMP}` on your machine. See longer explanation [here](https://github.com/kubernetes/kubernetes/pull/16287#discussion_r43333840).

The tl;dr is that when you call `docker run -v PATH:/containerpath ...`, the `PATH` must be resolvable on the host. To have the above command write output that you can then read here, you have to pass in the host-resolvable path (i.e., use REPO_DIR), and then read that output using the local path (*without* REPO_DIR).